Assets/TabMasterScript.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine;
+
+public class TabMasterScript : MonoBehaviour
+{
+    public tab1 tab_one;
```

Comment:
I'm assuming these are public because it makes it easier to drag them in, we will eventually move this to a register system where the `Tab` will register in a `TabSet` as it is the `Tab` classes's "responsibility" to tell the `TabManager` that it exists